core/src/main/resources/hudson/logging/LogRecorder/configure.jelly

```diff
@@ -27,9 +27,16 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-<l:layout>
+<l:layout permission="${app.ADMINISTER}" title="${%Configure Log Recorder}">
   <st:include page="sidepanel.jelly" />
   <l:main-panel>
+    <h1>${%Configure log recorder} <t:help iconSize="large" href="https://jenkins.io/redirect/log-recorders" /></h1>
```

Comment:
Same redirect as https://github.com/jenkinsci/jenkins/blob/ed80d24429a112f21fefaa09e39953c625628646/core/src/main/resources/hudson/logging/LogRecorderManager/index.jelly#L35 seems to make sense.